Configuration/PyReleaseValidation/scripts/runTheMatrix.py

```diff
@@ -56,7 +56,9 @@ def runSelected(opt):
                      136.731, #2016B Photon data
                      140.53, #2011 HI data
                      1330, #Run2 MC Zmm
-                     135.4 #Run 2 Zee ttbar
+                     135.4, #Run 2 Zee ttbar
+                     11024.0, #2023LReco ttbar (w/ HGCal)
+                     11224.0  #2023GReco ttbar (flat tracker, Run2 calo)
```

Comment:
need to decide if we want it here (for everyone running the short matrix) or in 
https://github.com/cms-sw/cms-bot/blob/master/cmssw-pr-test-config for jenkins jobs
Note that 2017 wflows are only in jenkins extras (10021.0,10024.0 )
If here, it makes more sense then to put 10021.0 and 10024.0  here as well